### DISCUSSION TITLE
index.qmd in book projects

### DISCUSSION THREAD

**Discussion by fredguth** [quarto-dev/quarto-cli#3047] [Q&A] [answered]
I am building a Quarto Project type that inherits from book.  The idea is to build a book and a website for the book at the same time. 

I don't know what to do with `index.qmd` which is necessary for the website but not for the pdf. I couldn't find a way to ignore index.qmd for pdf format. 

I tried:
``` index.qmd
:::{.content-hidden when-format="html"}:::
# Preface
:::
:::{.content-hidden when-format="pdf"}:::
# Welcome
:::
```
but `Preface` appears as the `page` title in the website top level ToC (error in website).

If I invert:
``` index.qmd
:::{.content-hidden when-format="pdf"}:::
# Welcome
:::
:::{.content-hidden when-format="html"}:::
# Preface
:::
```
now, ToC shows Welcome that should be ignored by pdf. ¯\_(ツ)_/¯

I guess the problem here has something to do with the fact that Welcome is the first top level section despite being inside a content-hidden block. Welcome is the `title` of the page, not `Preface`.

**Comment by dragonstyle** [comment, marked as answer]
Books are special, unfortunately, since we do special processing on titles (we capture the first H1 as the title of the page). The root issue is tracked here:

https://github.com/quarto-dev/quarto-cli/issues/1108
